.github/local-workflow-tech.md

```diff
@@ -17,25 +17,16 @@ Technicalities underlying the fast-pace local workflow
 - Switch to the branch corresponding to the robot (e.g., `devel-ergoCubSN???`).
 
 ## 🔲 Set the default committer
-- Fine-grained PAT can be scoped to work on a specified organization and only on a specified repository.
-- To store credentials locally, one needs to issue the following, once the fork is cloned locally:
+- In order to push to the ``icub-tech`` fork without password, it is necessary to get a PAT from the maintainer. Then, issue the following commands:
   ```console
   git config --local user.name "icub-tech-iit-bot"
   git config --local user.email "icub-tech@iit.it"
-  git config --local credential.https://github.com/icub-tech-iit/robots-configuration.git.helper store
+  PAT=<insert the PAT here>
+  git remote set-url icub-tech https://x-access-token:${PAT}@github.com/icub-tech-iit/robots-configuration.git
```

Comment:
```suggestion
  PAT=<insert the PAT here> git remote set-url icub-tech https://x-access-token:${PAT}@github.com/icub-tech-iit/robots-configuration.git
```

Otherwise, we need to use `export`.